vendor/github.com/OpenBazaar/multiwallet/client/blockbook/client.go

```diff
@@ -551,6 +562,12 @@ func (i *BlockBookClient) setupListeners() error {
 	i.listenLock.Lock()
 	defer i.listenLock.Unlock()
 
+	// Add stored watch addresses to listenQueue if there are any
+	i.listenQueue = append(i.listenQueue, i.listenAddrs...)
+
+	i.listenLock.Lock()
+	defer i.listenLock.Unlock()
```

Comment:
Remove this duplicate. `Lock` and deferred `Unlock` are already on L562 above.

Reply:
Fixed